pkg/stats/int-stats.go

```diff
@@ -0,0 +1,170 @@
+package stats
+
+import (
+	"math"
+	"sort"
+
+	"find-bin-width/pkg/xmath"
+	"find-bin-width/pkg/xtype"
+	"find-bin-width/pkg/xutil"
+)
+
+func calculateIntegerStats(values []int64) Stats {
+	result := sharedCalcIntegerStats(values)
+
+	var lq NullablePrimitive[float64]
+	var uq NullablePrimitive[float64]
+
+	if len(values) > 3 {
+		lq = NewNullableFloat(xmath.LowerQuartile(values))
+		uq = NewNullableFloat(xmath.UpperQuartile(values))
+	}
+
+	return stats[float64]{
+		min:           float64(result.min),
+		max:           float64(result.max),
+		binWidth:      float64(result.binWidth),
```

Comment:
nit: just noting that some of these things arent properly stats..

Reply:
renamed struct to `summary`